go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/hyperledger-labs/yui-relayer
 
-go 1.20
+go 1.21
```

Comment:
Please keep this at v1.20 since all the modules that depend on yui-relayer would need to be upgraded.
Only upgrade the components used in CI.

Reply:
Revert golang version
https://github.com/hyperledger-labs/yui-relayer/pull/102/commits/5a141cc3e72ae9376e6d9c90d97639590e5f3b47